.github/workflows/on_pull_request.yml

```diff
@@ -53,6 +53,8 @@ jobs:
           gem install bundler
           bundle config path vendor/bundle
           bundle install --jobs 4 --retry 3
+          bundle update fastlane
```

Comment:
Is this for some warnings I noticed, that we aren't on the latest version?

Reply:
yep, introduced while tinkering, I don't think this is required though. Gonna see if build time can be reduced by removing this.